content/en/core/releases/feature_releases.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Feature Releases"
+linkTitle: "Feature Releases"
+weight: 2
+no_list: true
+description: >
+    Feature Releases for the CHT Core Framework
+relatedContent: >
+    core/releases
+    contribute#are-you-a-partner-wondering-how-issues-are-prioritized
+    
+---
+
+The Product Team has two dedicated sub-teams (Allies and Care) who try to deliver much requested features at a pace that is faster than our release cycle. To this end, select partners working directly with Product Team can access what are called "Feature Releases". 
```

Comment:
I think Care should be Care Teams but if this has officially changed then ignore me!

```suggestion
The Product Team has two dedicated sub-teams (Allies and Care Teams) who try to deliver much requested features at a pace that is faster than our release cycle. To this end, select partners working directly with Product Team can access what are called "Feature Releases". 
```

Reply:
agreed!